Products/LDAPUserFolder/LDAPUserFolder.py

```diff
@@ -1234,7 +1237,7 @@ def getLDAPSchema(self):
     @security.protected(manage_users)
     def getSchemaDict(self):
         """ Retrieve schema as list of dictionaries """
-        all_items = sorted(self.getSchemaConfig().values())
+        all_items = self.getSchemaConfig().values()
```

Comment:
Why did you remove the call to `sorted` here? If it failed bacause the return value of `values` is not a list then coerce it into a list. Leave the sorting.

Reply:
because sorted() generates  a TypeError:

TRACEBACK
"/home/dev/workspace/zopefoundation/Products.LDAPUserFolder/Products/LDAPUserFolder/LDAPUserFolder.py", line 1231, in getLDAPSchema
    raw_schema = self.getSchemaDict()
  File "/home/dev/workspace/zopefoundation/Products.LDAPUserFolder/Products/LDAPUserFolder/LDAPUserFolder.py", line 1240, in getSchemaDict
    all_items = sorted(self.getSchemaConfig().values())
TypeError: '<' not supported between instances of 'dict' and 'dict'

Reply:
Please replace the quick fix with something that reflects the intention:

`all_items = sorted(self.getSchemaConfig().values(), key=lambda x: x['ldap_name'])`

Reply:
done:
https://github.com/dataflake/Products.LDAPUserFolder/pull/3/commits/c45411be4b267bda997ded76e2c827e494d3cfc8
Took some time to guess the intention: you want to get the ldap schema representation  the administrator has configured in Zope sorted alphabetically by "LDAP Attribute Name" [ldap_names:list]. 
This definitly makes sense. 
Thank you.

Reply:
That method is used (among other places) to get the schema items that show on the Schema tab. The sorting here is not su much about a specific item to sort on, just to make sure that the ordering is stable and doesn't change every time I click on the Schema tab.